docs/checklist.md

```diff
@@ -0,0 +1,55 @@
+# Checklist for a smooth package review
+
+We recommend going through this list and making sure you can tick all the boxes
+before submitting your package for review.
+
+The following rules are checked automatically in CI when you open a pull request,
+by [`typst-package-check`][check] (which you can run locally too).
+
+- [ ] The package manifest (`typst.toml`) is valid (no syntax error, and follows
+  [the defined schema][manifest]).
+- [ ] The package can be imported without errors. In particular, they should not
+  contain any syntax errors. It does not mean that should not contain bugs or
+  unexpected behaviors, since it is impossible to tell for sure that they will
```

Comment:
```suggestion
  contain any syntax errors. This does not mean that they should not contain bugs or
  unexpected behaviors, since it is impossible to tell for sure whether they will
```